host/create.c

```diff
@@ -1493,6 +1500,8 @@ oe_result_t oe_create_enclave(
 
         free(enclave->ecalls);
         free(enclave);
+        enclave->ocalls = NULL;
```

Comment:
accessing memory after freeing

Reply:
Good catch. What was I thinking?

Reply:
Fixed.

Reply:
I thought you put this in to make sure I was paying attention :)